dss/api/bundles/__init__.py

```diff
@@ -132,6 +133,40 @@ def get(
         response.headers['Link'] = link
         return response
 
+@dss_handler
+def enumerate(replica: str, prefix: str = 'bundles/', token: str = None,
```

Comment:
`token: str` should be `token: typing.Optional[str]` since it can take on a None value